WcaOnRails/spec/support/database_cleaner.rb

```diff
@@ -1,22 +1,23 @@
 RSpec.configure do |config|
+  reference_tables_to_keep = %w(Countries Continents Events Rounds Formats teams)
   config.before(:suite) do
-    DatabaseCleaner.clean_with(:truncation)
+    DatabaseCleaner.clean_with(:truncation, except: reference_tables_to_keep)
+    reference_tables_to_keep.each do |table|
+      ActiveRecord::Base.connection.execute("TRUNCATE #{table};")
+      load "#{Rails.root}/db/seeds/#{table.underscore}.seeds.rb"
```

Comment:
I was just thinking we'd move that code to `PopulateTestDb.rb `. Nothing fancy at all. Shouldn't take many minutes.

If it's used somewhere else, either that code could use `PopulateTestDb.rb `, or it's not worth messing with this.